src/start-sqm

```diff
@@ -16,6 +16,15 @@ STATE_FILE="${SQM_STATE_DIR}/${IFACE}.state"
 
 check_state_dir
 
+# log file for the most recent sqm instance start
+if [ "$SQM_DEBUG" -eq "1" ] ; then
+    SQM_DEBUG_LOG=${SQM_START_LOG}
+    OUTPUT_TARGET=${SQM_DEBUG_LOG}
+    echo "$(date): start-sqm: Log for interface ${IFACE}"  > ${OUTPUT_TARGET}
```

Comment:
Missing quotes.

Also, better to put the datestamp at the end of the line, so it always starts with the same string (for grepping)

Reply:
Great, will do.

Reply:
Fixed, not 100% sure about the grep argument, but I changed it anyway, I really just want to have this information and do not care where exactly the date hides ;)